yaksh/send_emails.py

```diff
@@ -58,3 +61,30 @@ def send_user_mail(user_mail, key):
         success = False
 
     return success, msg
+
+def send_bulk_mail(subject, email_body, recipients, attachments):
+    try:
+        text_msg = "Yaksh"
```

Comment:
This variable is required because django send mail does not accept email message as None.
Message from textarea is received as html message which cannot be passed as a plain text.